src/parser/csql_grammar.y

```diff
@@ -27854,3 +27789,38 @@ pt_ct_check_select (char* p, char *perr_msg)
    sprintf(perr_msg, "Only SELECT statements are supported.");
    return false;
 }
+
+static PT_NODE*
+get_string_literal_node(const char* str, const int opt_str_size) {
```

Comment:
naming: how about get_sting_literal_node () -> pt_create_string_literal_node_w_charset_coll ()?

Reply:
OK. thank you